packages/nvidia_nat_atif/src/nat/atof/codec.py

```
@@ -0,0 +1,199 @@
+
+from __future__ import annotations
+
+from typing import Any
+from typing import Literal
+
+from pydantic import BaseModel
+from pydantic import ConfigDict
+from pydantic import Field
+
+# ---------------------------------------------------------------------------
+# Request-side types (Section 5.1–5.6)
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)